_layouts/front.html

```diff
@@ -98,18 +98,49 @@ <h3>Mentorship</h3>
                 <h2 class="section-heading">Get Involved!</h2>
                 <hr class="dark">
                 <p class="text-muted">
-                    We are always looking for high school and college students to help us support our community! 
-                    Whether you are good at art, like to post on social media, have great scores on tests and in school, 
-                    want to share your college application journey with others, or like writing articles, 
+                    We are always looking for high school and college students to help us support our community!
+                    Whether you are good at art, like to post on social media, have great scores on tests and in school,
+                    want to share your college application journey with others, or like writing articles,
                     there's always a position for you on our team!</p>
                 <a href="/get-involved" class="btn btn-primary btn-xl">Learn More!</a>
             </div>
         </div>
     </div>
   </section>
 
+  <section class="bg-light" id="footer">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-3">
+          <h3 class="section-heading">Social Media</h3>
+          <ul class="list-unstyled" id="footer-links">
+            <li><a href="https://discord.myeduvisa.org">Discord</a></li>
```

Comment:
Is it possible to make Discord a standalone section with a large button (text something like "Join Us On Discord!", similar to the header)? I think it'd be nice to have it separately to emphasize the community aspect.

Reply:
Actually in general, it it possible to make the social media column into icons? you should be able to find icons for every platform on Font Awesome (e.g. https://fontawesome.com/icons/facebook-square?style=brands) which we're using (check out the discord icon in the header for an idea about how to use this)